platform-server-kernel/src/main/java/uno/cod/platform/server/core/dto/challenge/ChallengeCreateDto.java

```diff
@@ -10,6 +10,9 @@
     private String name;
 
     @NotNull
+    private Long organizationId;
+
```

Comment:
What does public mean anyway? If it means that anybody can do the challenge (without invitation), I am sure companies will like that.
How do you interpret it so that you assume they will not like it?